server/build.gradle

```diff
@@ -34,39 +34,39 @@ publishing {
 
 archivesBaseName = 'elasticsearch'
 
-// we want to keep the JDKs in our IDEs set to JDK 8 until minimum JDK is bumped to 9 so we do not include this source set in our IDEs
+// we want to keep the JDKs in our IDEs set to JDK 11 until minimum JDK is bumped to 17 so we do not include this source set in our IDEs
```

Comment:
s/17/12/ ?

Reply:
I think it should be 18? The next LTS release?

Reply:
Yeah, I was going for the next LTS release which I had in my head was 17. I think that's right though? JDK LTS releases every three years, 11 was in September 2018 so the next one would be September 2021 which would be:
 - 13 September 2019
 - 14 March 2020
 - 15 September 2020
 - 16 March 2021
 - 17 September 2021